src/components/ImageScroller.js

```diff
@@ -129,7 +165,15 @@ export default function ImageScroller(props) {
                 height={192}
                 className={classes.image}
                 fixed
+                style={{ transform: `rotate(${rotation}deg)` }}
               />
+              <IconButton
```

Comment:
We should only show the rotation button on (and apply to the rotation to) the selected image.
Otherwise it looks a bit messy with multiple images:

https://user-images.githubusercontent.com/5558838/128629855-3437f0e5-b1e6-40f3-bf75-1d3d36c258c6.mov